details/input.md

```diff
@@ -0,0 +1,172 @@
+# Interoperable Private Attribution wire format
+
+
+This documents provides clarification on the format IPA parties use to submit queries. 
+
+## Query
+
+IPA query consists of a mix of source and trigger events obtained from one or many source and trigger websites. 
+**Query size** is determined by the number of events included within a single query request. 
+
+It is desirable for report collectors to submit large queries as it brings more utility and saves cost, 
+therefore it makes sense to optimize the query format on the wire.
+
+The following sections propose the format that is space-optimized at the expense of being more complicated to assemble. 
+
+## Assumptions
+
+* Report collector use HTTP over TLS to send queries to helper party networks.
```

Comment:
"over" here means that the one protocol layer (HTTP) runs on top of the other (TLS).  It doesn't mean "instead of", even though that is another meaning that "over" can take, it isn't the usual assumption in this context.

Reply:
oh ok.. maybe we should reword it to say "on top of" TLS to avoid confusion?

Reply:
heh, I never read it that way - too used to this expression. I agree that it does sound confusing, so I'll just use HTTPS instead.

Reply:
"HTTP over TLS" is the name of the protocol.  Or "HTTPS".  To call it something else would be far worse.

Reply:
Didnt know about that. How about we add a link to RFC for it : https://www.rfc-editor.org/rfc/rfc2818